client/src/components/Homepage.js

```diff
@@ -18,40 +19,91 @@ function Homepage({ searchText }) {
       });
   }
 
-  useEffect(loadFeed, [feedURL, searchText]);
+  function toggleSortOrder() {
+    setSortOrder(sortOrder === 'ASC' ? 'DESC' : 'ASC');
+  }
+
+  useEffect(loadFeed, [feedURL, searchText, sortOrder]);
+
+  useUpdateCheck(feed, searchText, sortOrder, 10000);
 
   return (
-    <form
-      className="flex flex-col max-w-screen-md mx-auto my-4 space-y-8 cursor-pointer"
-      onClick={() => console.log("TODO: Load this Post!")}
-    >
-      {feed.map((post) => (
-        <div
-          key={post.id}
-          className="flex rounded-xl border border-gray-300 max-h-60 overflow-hidden"
-        >
-          {post.image && (
-            <img className="h-full w-60" src={post.image} alt="Logo" />
+    <div className='flex flex-col max-w-screen-md mx-auto my-4 space-y-4'>
```

Comment:
change class name to 'w-full flex flex-col items-center my-4 space-y-4 mb-28' and then have each component define their widths.